src/crypto/tls/cfkem.go

```diff
@@ -29,38 +28,27 @@ import (
 	"github.com/cloudflare/circl/kem/hybrid"
 )
 
-// Either ecdheParameters or kem.PrivateKey
+// Either *ecdh.PrivateKey or *kemPrivateKey
 type clientKeySharePrivate interface{}
 
+type kemPrivateKey struct {
+	secretKey kem.PrivateKey
+	curveID   CurveID
+}
+
 var (
-	X25519Kyber512Draft00 = CurveID(0xfe30)
-	X25519Kyber768Draft00 = CurveID(0xfe31)
-	P256Kyber768Draft00   = CurveID(0xfe32)
-	invalidCurveID        = CurveID(0)
+	X25519Kyber512Draft00    = CurveID(0xfe30)
```

Comment:
Did you consider dropping Kyber512? My understanding is that (1) it's not currently in use anywhere and (2) we're not going to need it in the future.

Reply:
We deployed Kyber512 alongside Kyber768 to get data on whether it's more deployable or not.